what-our-users-say.html

```diff
@@ -242,6 +242,14 @@ <h5 style="font-weight: normal">Let's continue to build our community together.<
                     </a>
                 </div>
                 <div class="col-sm-12 col-md-6">
+                    <a class="card-2">
+                    <div class="row" style="padding:10px">
+                        "At Codewave, we help SME’s and Enterprises across the globe, modernise their existing applications or build robust new innovative digital initiatives from the ground up and Loopback has "got our back" on more than one occasions, bringing rapid innovation without compromising on security or scalability."
```

Comment:
Minor edits:
```suggestion
                        "At Codewave, we help SME’s and Enterprises across the globe, modernise their existing applications and build robust new innovative digital initiatives from the ground up and LoopBack has "got our back" on more than one occasions, bringing rapid innovation without compromising on security or scalability."
```

Reply:
@dhmlau thanks fixed. And the new entry is also added.